openassessment/xblock/static/js/spec/lms/oa_staff_area.js

```diff
@@ -427,4 +445,110 @@ describe('OpenAssessment.StaffAreaView', function() {
             expect($staffInfoPanel).toHaveClass('is--hidden');
         });
     });
+
+    describe('Grade Available Responses', function() {
```

Comment:
@andy-armstrong can you add coverage for the display of number ungraded and number in progress?